test/common/testlib.py

```diff
@@ -1231,10 +1231,10 @@ class MachineCase(unittest.TestCase):
     image = testvm.DEFAULT_IMAGE
     libexecdir = None
     runner = None
-    machine = None
+    machine: testvm.Machine
```

Comment:
This is a curious syntax -- so it exists purely to declare the type now, but does not actually "create" the member?